app/test/unit/infer-comparison-branch-test.ts

```diff
@@ -35,29 +35,16 @@ function createTestGhRepo(
   defaultBranch: string | null = null,
   parent: GitHubRepository | null = null
 ) {
-  if (owner.indexOf('/') !== -1) {
-    throw new Error(
-      'Providing a slash in the repository name is no longer supported, please update your test'
-    )
-  }
-
-  const cloneURL = `https://github.com/${owner}/my-cool-repo.git`
-
-  return new GitHubRepository(
-    name,
-    new Owner('', '', null),
-    null,
-    false,
-    '',
-    `${
+  return gitHubRepoFixture({
+    owner: owner,
+    name: 'my-cool-repo',
+    defaultBranch: `${
       defaultBranch !== null && defaultBranch.indexOf('/') !== -1
         ? defaultBranch.split('/')[1]
         : defaultBranch
     }`,
-    cloneURL,
-    'write',
```

Comment:
Ah okay, looks like we default to having write permissions so it's not necessary to specify `write` 👍 -- https://github.com/desktop/desktop/blob/88e9777a0a847eeaff3aa8840e2d745130160bdd/app/src/models/github-repository.ts#L58-L65